microceph/ceph/config.go

```diff
@@ -367,3 +370,18 @@ func getMonitorAddresses(configs map[string]string) []string {
 	}
 	return monHosts
 }
+
+// formatIPv6 returns a slice in which all IPv6 addresses are formatted with square brackets.
+func formatIPv6(addrs []string) []string {
+	formatted := []string{}
+	for _, addr := range addrs {
+		ip := net.ParseIP(addr)
+		if ip != nil && strings.Contains(addr, ":") {
```

Comment:
should the ip be added to the list if parsing it returned `nil`?

Reply:
Hi @lmlg, `net.ParseIP` will return nil if the IPv6 address is already enclosed in square brackets, so we would fail to add the addresses in this case. If it is not possible for this to happen, let me know and I'll make this change.

Reply:
If we're certain that `ParseIP` can only fail for that reason, then it's fine by me.